### PR TITLE
[19/N] Fix clang-tidy warnings in jit

### PR DESCRIPTION
**Pull request by cyyever**
Follows  #132963

### CODE REVIEW

File: torch/csrc/jit/mobile/file_format.h [hunk 1]
```diff
@@ -159,7 +153,6 @@ static inline std::tuple<std::shared_ptr<char>, size_t> get_file_content(
   return std::make_tuple(data, size);
 }
 
-// NOLINTNEXTLINE(facebook-hte-NamespaceScopedStaticDeclaration)
```
Comment (Skylion007) [inline torch/csrc/jit/mobile/file_format.h:162]: Ugh, I think these are internal clang-tidy checks? We can't  shouldn't just disable them.

**Review by Skylion007 — CHANGES_REQUESTED** [review]
don't disable facebook clang-tidy lint. It will break internal lints

**Comment by cyyever** [comment]
> don't disable facebook clang-tidy lint. It will break internal lints

Turned them from 'static inline' to inline, which should fix the lints.

**Review by Skylion007 — APPROVED** [review]

**Pull request closed without merging**